client/jetpack-onboarding/steps/woocommerce.jsx

```diff
@@ -19,7 +20,16 @@ class JetpackOnboardingWoocommerceStep extends React.PureComponent {
 			"We'll set you up with WooCommerce for all of your online selling needs."
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		return (
+			<Fragment>
+				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+
+				<div className="steps__button-group">
+					<Button primary>{ translate( 'Yes, I am' ) }</Button>
+					<Button>{ translate( 'Not right now' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **98** times:
`translate( 'Right Now' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-onboarding-woocommerce-step-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).